ml-package-versions.yml

```diff
@@ -0,0 +1,162 @@
+
+  models:
+    minimum: "1.4.0"
+    maximum: "1.7.0"
+    requirements:
+      # https://github.com/pytorch/vision#installation
+      "== 1.4.0": ["torchvision==0.5.0", "scikit-learn"]
+      "== 1.5.0": ["torchvision==0.6.0", "scikit-learn"]
+      "== 1.5.1": ["torchvision==0.6.1", "scikit-learn"]
+      "== 1.6.0": ["torchvision==0.7.0", "scikit-learn"]
+      "> 1.6.0": ["torchvision", "scikit-learn"]
+    run: |
+      pytest tests/pytorch/test_pytorch_model_export.py --large
+
+  autologging:
+    minimum: "1.4.0"
+    maximum: "1.7.0"
+    requirements:
+      "== 1.4.0": ["torchvision==0.5.0", "pytorch-lightning", "scikit-learn"]
+      "== 1.5.0": ["torchvision==0.6.0", "pytorch-lightning", "scikit-learn"]
+      "== 1.5.1": ["torchvision==0.6.1", "pytorch-lightning", "scikit-learn"]
+      "== 1.6.0": ["torchvision==0.7.0", "pytorch-lightning", "scikit-learn"]
+      "> 1.6.0": ["torchvision", "scikit-learn"]
+    run: |
+      pytest tests/pytorch/test_pytorch_model_export.py --large
+
+tensorflow:
+  package_info:
+    pip_release: "tensorflow"
+    pip_dev: |
+      pip install tf-nightly
+
+  models:
+    minimum: "2.0.0"
+    maximum: "2.3.1"
+    run: |
+      pytest tests/tensorflow/test_tensorflow2_model_export.py --large
+
+  autologging:
+    minimum: "2.0.0"
+    maximum: "2.3.1"
+    requirements:
+      "< 2.2": ["h5py<3.0"]
+    run: |
+      pytest --verbose tests/tensorflow_autolog/test_tensorflow2_autolog.py --large
+
+  tf1:
+    minimum: "1.15.4"
+    maximum: "1.15.4"
+    # Avoid fetching versions up until the latest
+    until_latest: False
+    # Avoid running tests on `tf-nightly`
+    include_dev: False
+    requirements:
+      [
+        "keras==2.2.5",
+        "scikit-learn",
+        "h5py<3.0.0",
+        "pyspark==2.4.0",
+        "pyarrow",
+      ]
+    run: |
+      pytest tests/tensorflow/test_tensorflow_model_export.py --large
+      pytest tests/tensorflow_autolog/test_tensorflow_autolog.py --large
+      pytest tests/keras --large
+      pytest tests/keras_autolog --large
+
+keras:
+  package_info:
+    pip_release: "keras"
+    pip_dev: |
+      # In keras >= 2.4.0, `keras` simply redirects to `tensorflow.keras`:
+      # https://github.com/keras-team/keras#multi-backend-keras-and-tfkeras
+      pip install tf-nightly 'keras>=2.4.0'
+
+  models:
+    minimum: "2.3.1"
+    maximum: "2.4.3"
+    requirements:
+      "== 2.3.1":
+        ["tensorflow==2.2.1", "scikit-learn", "pyspark==2.4.0", "pyarrow"]
+      "> 2.3.1": ["tensorflow", "scikit-learn", "pyspark==2.4.0", "pyarrow"]
+    run: |
+      pytest tests/keras/test_keras_model_export.py --large
+
+  autologging:
+    minimum: "2.3.1"
+    maximum: "2.4.3"
+    requirements:
+      "== 2.3.1": ["tensorflow==2.2.1"]
+      "> 2.3.1": ["tensorflow"]
+    run: |
+      pytest --verbose tests/keras_autolog/test_keras_autolog.py --large
+
+xgboost:
+  package_info:
+    pip_release: "xgboost"
+    pip_dev: |
```

Comment:
`pip_dev` is a misnomer here, since the installation instructions don't involve `pip`. Can we call this field `install_dev` instead? That way, we have two fields: `pip_release` and `install_dev`.

Reply:
makes sense. I'll fix this

Reply:
Done!